webpack.config.js

```diff
@@ -21,17 +21,35 @@ dotenv.config({path: '.env.default'});
  */
 module.exports = (env = {NODE_ENV: process.env.NODE_ENV || 'production'}) => ({
   entry: {
-    webex:
-      env && env.NODE_ENV === 'development'
-        ? `${path.resolve(__dirname)}/packages/webex/src/index.js`
-        : './packages/webex',
-    meetings: `${path.resolve(__dirname)}/packages/webex/src/meetings.js`,
-    calling: `${path.resolve(__dirname)}/packages/webex/src/calling.js`,
+    webex: {
+      import:
+        env && env.NODE_ENV === 'development'
+          ? `${path.resolve(__dirname)}/packages/webex/src/index.js`
+          : './packages/webex',
+      library: {
+        name: 'Webex',
+        type: 'umd',
+      },
+    },
+    meetings: {
+      import: `${path.resolve(__dirname)}/packages/webex/src/meetings.js`,
+      library: {
+        name: 'Webex',
+        type: 'umd',
+      },
+    },
+    calling: {
+      import: `${path.resolve(__dirname)}/packages/webex/src/calling.js`,
+      library: {
+        name: 'Calling',
+        type: 'umd',
+        export: 'default',
```

Comment:
why do we need to add export here specifically for calling only

Reply:
So, since the calling.js is es6 module, we cannot do `export Calling` - this will throw syntax error.

What we could do is `export default Calling` which equals to `module.exports.default = Calling`

In this scenario, if we do not add this config, webpack will export the Calling class in the global object inside the library `Calling`. So you would have to access it like `new Calling.Calling()`

Whereas, when you add this, whatever is exported as default will be assigned to the library `Calling` itself.

- Read more about webpack library config here -> [webpack:output.library](https://webpack.js.org/configuration/output/#outputlibrary)
- Read more about the webpack library export config here -> [webpack:output.library.export](https://webpack.js.org/configuration/output/#outputlibraryexport)